clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/AppEngineUtils.groovy

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.appengine.deploy
+
+import com.google.api.services.appengine.v1.model.Service
+import com.google.api.services.appengine.v1.model.Version
+import com.netflix.spinnaker.clouddriver.appengine.security.AppEngineNamedAccountCredentials
+import com.netflix.spinnaker.clouddriver.data.task.Task
+
+class AppEngineUtils {
+  static List<Version> queryAllVersions(String project,
+                                        AppEngineNamedAccountCredentials credentials,
+                                        Task task,
+                                        String phase) {
+    task.updateStatus phase, "Querying all versions for project $project..."
+    def services = queryAllServices(project, credentials, task, phase)
+
+    services.collect {
```

Comment:
I think our de-factor style is to keep the closure parameters on the same line as the `{`